source/titfront/src/components/PythonShell.tsx

```
@@ -0,0 +1,181 @@
+    // Cmd+K clears the console on macOS. Let's do the same.
+    if (event.key === "k" && (event.ctrlKey || event.metaKey)) {
+      event.preventDefault();
+      setFirstCommand(commands.length);
+      historyIndex = undefined;
+      return;
+    }
+
+    // Browse the history.
+    const setInput = (value: string) => {
+      // Update the input.
+      inputRef.current!.textContent = value;
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)